File: Python/MyNumberLookBigInThis/MySolution.py
def narcissistic( value ):
    result = 0
    for digit in str(value):
        result += int(digit)**len(str(value))
    return True if (value == result) else False

# Original kata: https://www.codewars.com/kata/5287e858c6b5a9678200083c
# My solution: https://www.codewars.com/kata/reviews/55db4cb62ede87b90600007e/groups/5b77214cb6c53e5a53000e7c

File: PHP/PyramidArray/MySolution.php
<?php

function pyramid($n) {
  $result = [];
  for ($i=1; $i <= $n ; $i++) { 
      $result[] = array_fill(0, $i, 1);
  }
  return $result;
}

// Original kata: https://www.codewars.com/kata/515f51d438015969f7000013
// My solution: https://www.codewars.com/kata/reviews/5dacc032564ff100010c3f44/groups/5dc7d03c2d55c300010eaa28
